ui/src/app/projects/projects.component.ts

```diff
@@ -94,6 +95,10 @@ export class ProjectsComponent implements OnInit {
 
   navigateJobs() {
     let extras = {queryParams: {q: URLSearchParamsUtils.encodeURLSearchParams({extensions: {projectId: this.projectsControl.value}})}};
-    this.router.navigate(['jobs'], extras);
+    if (environment.dashboardEnabled) {
+      this.router.navigate(['dashboard'], extras);
```

Comment:
Have you thought about the URL scheme? I'm thinking we may just want `projectId=<project>`

Reply:
I thought we can parse that 'extra' thing when handling the navigation. It is not that big.

Reply:
I think the only filter we would be supporting here is projectId (and possibly a query param for the time range), so I wouldn't support the more generic "?q=<encoded_search_query>" that we have on the job list page.